benches/bench.rs

```diff
@@ -27,18 +27,19 @@ pub fn read_section(section: &str) -> Vec<u8> {
 fn bench_parsing_debug_abbrev(b: &mut test::Bencher) {
     let debug_info = read_section("debug_info");
     let debug_info = DebugInfo::<LittleEndian>::new(&debug_info);
-    let unit = debug_info.units()
+    let unit = debug_info
+        .units()
         .next()
         .expect("Should have at least one compilation unit")
         .expect("And it should parse OK");
 
     let debug_abbrev = read_section("debug_abbrev");
 
     b.iter(|| {
-        let debug_abbrev = DebugAbbrev::<LittleEndian>::new(&debug_abbrev);
-        test::black_box(unit.abbreviations(debug_abbrev)
-            .expect("Should parse abbreviations"));
-    });
+               let debug_abbrev = DebugAbbrev::<LittleEndian>::new(&debug_abbrev);
+               test::black_box(unit.abbreviations(debug_abbrev)
+                                   .expect("Should parse abbreviations"));
+           });
```

Comment:
Bleh... I've tried to figure out how to fix this indentation behavior, haven't found anything that quite does the trick.